static/app/views/replays/utils/useReplayEvent.tsx

```diff
@@ -0,0 +1,180 @@
+import {useEffect, useState} from 'react';
+import type RRWebPlayer from 'rrweb-player';
+
+import {IssueAttachment} from 'sentry/types';
+import {Event} from 'sentry/types/event';
+import EventView from 'sentry/utils/discover/eventView';
+import {generateEventSlug} from 'sentry/utils/discover/urls';
+import RequestError from 'sentry/utils/requestError/requestError';
+import useApi from 'sentry/utils/useApi';
+
+export type RRWebEvents = ConstructorParameters<typeof RRWebPlayer>[0]['props']['events'];
```

Comment:
in the SDK I used: `import type { eventWithTime } from 'rrweb/typings/types';` (I think they're the same type?)

Reply:
@ryan953 We may want to alias this as like `RRWebEvent` or something in a future PR